phoenix-core/src/main/java/org/apache/phoenix/schema/PTableImpl.java

```diff
@@ -1008,6 +1030,11 @@ public final PName getTableName() {
         return tableName;
     }
 
+    @Override
+    public final PName getPhysicalTableName() {
```

Comment:
As mentioned elsewhere, why both the existing getPhysicalName and a new getPhysicalTableName? Can they be consolidated? Or at least named something more clear?

Reply:
It is a bit complicated. getPhysicalName is mostly used for views. getPhysicalTableName is mostly for non-views. Let me see what I can do.